index.html

```diff
@@ -254,66 +252,33 @@
       </div></div>
       <div id="main-content"></div>
       <div id="toast-container"></div>
-
-
-      <noscript id="deferred-styles">
-          <link rel="stylesheet" type="text/css" href="{{CDN_SERVICE}}/{{RELEASE}}/ogs.{{VERSION_DOTCSS}}">
-      </noscript>
-
       <script>
         try {
             var script = document.createElement('script');
             script.id = "ogs_score_estimator_script";
             try {
                 new WebAssembly.Memory({ initial: 0 });
                 console.log("WASM support detected, score estimator will be fast");
-                script.src = "/OGSScoreEstimator/OGSScoreEstimator-0.7.0.js";
+                script.src = "//localhost:8080/OGSScoreEstimator/OGSScoreEstimator-0.7.0.js";
             } catch (error) {
                 console.warn(error);
                 console.log("No WASM support detected, score estimator falling back to ASM.js mode");
-                script.src = "/OGSScoreEstimator/OGSScoreEstimator-0.7.0.asm.js";
+                script.src = "//localhost:8080/OGSScoreEstimator/OGSScoreEstimator-0.7.0.asm.js";
```

Comment:
Will this work in other environments? (i.e. user clients)

Reply:
Values are temporarily hardcoded for testing.